<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<class>GovernancePage</class>
<widget class="QWidget" name="GovernancePage">
 <property name="geometry">
  <rect>
   <x>0</x>
   <y>0</y>
   <width>629</width>
   <height>406</height>
  </rect>
 </property>
 <property name="windowTitle">
  <string>Form</string>
 </property>
 <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,1">
  <property name="spacing">
   <number>0</number>
  </property>
  <property name="leftMargin">
   <number>0</number>
  </property>
  <property name="topMargin">
   <number>0</number>
  </property>
  <property name="rightMargin">
   <number>0</number>
  </property>
  <property name="bottomMargin">
   <number>0</number>
  </property>
  <item>
   <widget class="QWidget" name="left" native="true">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>20</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="labelTitle">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignTop">
            <widget class="QLabel" name="labelSubtitle1">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>20</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_Content">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QScrollArea" name="scrollArea">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>1</verstretch>
                </sizepolicy>
               </property>
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Plain</enum>
               </property>
               <property name="lineWidth">
                <number>0</number>
               </property>
               <property name="horizontalScrollBarPolicy">
                <enum>Qt::ScrollBarAlwaysOff</enum>
               </property>
               <property name="sizeAdjustPolicy">
                <enum>QAbstractScrollArea::AdjustIgnored</enum>
               </property>
               <property name="widgetResizable">
                <bool>true</bool>
               </property>
               <widget class="QWidget" name="scrollAreaWidgetContents2">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>742</width>
                  <height>282</height>
                 </rect>
                </property>
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>1</verstretch>
                 </sizepolicy>
                </property>
                <layout class="QVBoxLayout" name="proposalGrid">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="sizeConstraint">
                  <enum>QLayout::SetDefaultConstraint</enum>
                 </property>
                 <property name="margin">
                  <number>0</number>
                 </property>
                </layout>
               </widget>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_20">
               <item>
                <widget class="QPushButton" name="UpdateButton">
                 <property name="text">
                  <string>Update Proposals</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_11">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Minimum</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="CreateProposalButton">
                 <property name="text">
                  <string>Create Proposals</string>
                 </property>
                </widget>
               </item>
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>20</number>
               </property>
               <item>
                <spacer name="horizontalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QFrame" name="frame_budget_info">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="next_superblock_label">
                <property name="text">
                 <string>Next super block:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1" alignment="Qt::AlignRight">
               <widget class="QLabel" name="next_superblock_value">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="blocks_before_super_label">
                <property name="text">
                 <string>Blocks to next super block:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1" alignment="Qt::AlignRight">
               <widget class="QLabel" name="blocks_before_super_value">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="time_before_super_label">
                <property name="text">
                 <string>Days to budget payout (estimate):</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1" alignment="Qt::AlignRight">
               <widget class="QLabel" name="time_before_super_value">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="alloted_budget_label">
                <property name="text">
                 <string>Allotted budget:</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1" alignment="Qt::AlignRight">
               <widget class="QLabel" name="alloted_budget_value">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="unallocated_budget_label">
                <property name="text">
                 <string>Budget left:</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1" alignment="Qt::AlignRight">
               <widget class="QLabel" name="unallocated_budget_value">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="masternode_count_label">
                <property name="text">
                 <string>Masternodes count:</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1" alignment="Qt::AlignRight">
               <widget class="QLabel" name="masternode_count_value">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <property name="rightMargin">
          <number>20</number>
         </property>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </item>
 </layout>
</widget>
<resources/>
<connections/>
</ui>
